app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -67,8 +67,10 @@ def calculate_line_7
       end
 
       def calculate_line_8
-        number_of_line_8_exemptions = number_of_true_checkboxes([@direct_file_data.is_primary_blind?,
-                                                                 @direct_file_data.is_spouse_blind?])
+        primary_disabled = @intake.primary_disabled == "yes"
```

Comment:
to avoid the possibility of typos, you can use the generated enum methods here (`@intake.primary_disabled_yes?`, along with `_no?` and `_unfilled?`) and elsewhere that you check against string literals

Reply:
So useful! Thanks for the heads up about the generated methods, Mike. Fixing before merge.